artemis-server/src/main/java/org/apache/activemq/artemis/core/postoffice/impl/PostOfficeImpl.java

```diff
@@ -829,7 +829,13 @@ public RoutingStatus route(final Message message,
 
       // TODO auto-create queues here?
       // first check for the auto-queue creation thing
-      if (bindings == null) {
+      if (bindings == null && !address.toString().equals("activemq.notifications")) {
```

Comment:
This seems a symptom of another issue instead of a proper fix.

it seems you are fixing a symptom of another issue. Accepting this fix will probably hide a real issue somewhere else.


Can you replicate this on a test.. so we can validate the real issue?